### ISSUE TITLE
peers-code-review Accessibility Issues 

### ISSUE THREAD

**Issue by Moathal** [PRATAP-KUMAR/My-Portfolio#7]
You have accomplished all requirements this is enough to show how committed you are 🥇 

Yet there are a few things if modified you will be the supreme coder 😃 

- For accessibility reasons, I think the title should be more descriptive like 'Pratap's Portfolio'. 🕵️‍♀️ 
- Kindly make sure alts attribute in img tag are descriptive  since it will be read by the screen reader for people with disability. 🤔 
- Consider sticking to semantic tags as it will help us with readability by using only one h1 tag. 🥺
